lib/generators/data_migration/templates/data_migration.template

```diff
@@ -0,0 +1,11 @@
+class <%= @class_name %>
```

Comment:
Because we don't have the same level of complexity around running these as normal schema migrations (up, down, rollback, etc) we don't need the extra ceremony and faff of defining a named class with an `up` method that gets called on the next line. We can literally have a `# put your Ruby code here` line after the `# Run me with` line and that is enough.